launcher/launcher.go

```diff
@@ -1213,3 +1213,29 @@ func deviceAllocateAlert(instance *runtimeInstanceInfo, device string, err error
 		},
 	}
 }
+
+func instanceInfoEqual(info1, info2 *aostypes.InstanceInfo) bool {
+	if info1.InstanceIdent != info2.InstanceIdent ||
```

Comment:
You can first compare DNSServers then assigns nil and then just compare structures: info1 != info2.

Reply:
in this case i got an error 
invalid operation: info1 != info2 (struct containing aostypes.NetworkParameters cannot be compared)

Reply:
Can we use DeepEqual here as well?

Reply:
But I thought we only use DeepEqual in tests?